### PR TITLE
FLW-868 Make OnFinality nodes default one

### PR DESCRIPTION
**Pull request by ef1rspb**

### CODE REVIEW

File: fearless/Common/Extension/Storage/Storage+Identifiable.swift [hunk 1]
```diff
@@ -6,7 +6,7 @@ extension AccountItem: Identifiable {
 }
 
 extension ConnectionItem: Identifiable {
-    var identifier: String { url.absoluteString }
+    var identifier: String { (url.scheme ?? "") + "://" + (url.host ?? "") }
```
Comment (antonkhvorov) [inline fearless/Common/Extension/Storage/Storage+Identifiable.swift:9]: what about IPs? I think that's the reason why before it was an absoluteString

File: fearless/Common/Extension/Model/ConnectionItem+Default.swift [hunk 2]
```diff
@@ -4,22 +4,22 @@ import IrohaCrypto
 extension ConnectionItem {
     static var defaultConnection: ConnectionItem {
```
Comment (antonkhvorov) [inline fearless/Common/Extension/Model/ConnectionItem+Default.swift:5]: Do we still need that? @ERussel 
Before we had a Kusama account as a default, thus this was probably used, but now users can create an account in any network as their first account, so maybe we can just use supportedConnections?

File: fearless/Common/Extension/Storage/Storage+Identifiable.swift (hunk 1 above)
Comment (ERussel) [reply fearless/Common/Extension/Storage/Storage+Identifiable.swift:9]: Didn't get why we need to change that?

File: fearless/Common/Extension/Model/ConnectionItem+Default.swift (hunk 2 above)
Comment (ERussel) [reply fearless/Common/Extension/Model/ConnectionItem+Default.swift:5]: Yep, let's leave as it is for now

**Review by ERussel — APPROVED** [review]

**Pull request merged**